thoth/adviser/exceptions.py

```diff
@@ -102,3 +108,19 @@ def __init__(self, *args: Any, unresolved: List[str]) -> None:
         """Capture unresolved dependencies in this exception."""
         super().__init__(*args)
         self.unresolved = unresolved
+
+    def to_dict(self) -> Optional[Dict[str, str]]:
+        """Convert unresolved dependencies exception to the user."""
+        return {
+            "ERROR": f"No dependencies found for {', '.join(f'{dep!r}' for dep in self.unresolved)}; "
```

Comment:
can we say why we have not found any dependencies?

Reply:
Yes, I think so :+1:

Reply:
good, lets do it :)

Reply:
Done it in b30d64 :)